components/director/internal/domain/scenarioassignment/resolver_test.go

```diff
@@ -72,7 +72,9 @@ func TestResolverCreateAutomaticScenarioAssignment(t *testing.T) {
 		fetcherSvc := &automock.TenantFetcher{}
 		fetcherSvc.On("FetchOnDemand", mock.Anything).Return(testErr).Once()
 
-		sut := scenarioassignment.NewResolver(transact, nil, nil, nil, fetcherSvc)
+		formationSvc := &automock.FormationService{}
```

Comment:
You can pass nil to the constructor if it is unused in this particular test.

Valid for tests below as well.

Reply:
Will do, I removed the unused ones in the service_test.go, but initially I added all new mocks to see if and when they are used in each test instead of a nil exception and I forgot to remove these here. Thanks.